doc/DataPlaneCodeGeneration/AzureSDKCodeGeneration_DataPlane_Quickstart.md

```diff
@@ -66,15 +71,16 @@ eng/scripts/automation/Invoke-DataPlaneGenerateSDKPackage.ps1 -service <servicen
 e.g.
 
 ```powershell
-pwsh /home/azure-sdk-for-net/eng/scripts/automation/Invoke-DataPlaneGenerateSDKPackage.ps1 -service sample -namespace Azure.Template.Sample -sdkPath /home/azure-sdk-for-net -inputfiles https://github.com/Azure/azure-rest-api-specs/blob/73a0fa453a93bdbe8885f87b9e4e9fef4f0452d0/specification/webpubsub/data-plane/WebPubSub/stable/2021-10-01/webpubsub.json -securityScope https://sample/.default
+pwsh /home/azure-sdk-for-net/eng/scripts/automation/Invoke-DataPlaneGenerateSDKPackage.ps1 -service sample -namespace Azure.Template.Sample -sdkPath /home/azure-sdk-for-net -inputfiles https://github.com/Azure/azure-rest-api-specs/blob/73a0fa453a93bdbe8885f87b9e4e9fef4f0452d0/specification/webpubsub/data-plane/WebPubSub/stable/2021-10-01/webpubsub.json -readme https://github.com/Azure/azure-rest-api-specs/blob/73a0fa453a93bdbe8885f87b9e4e9fef4f0452d0/specification/webpubsub/data-plane/readme.md -securityScope https://sample/.default
```

Comment:
I don't think we want to pass both `-inputfiles` and `-readme` at the same time.  If `-readme` is passed, it will give us the list of input files.

Reply:
From my test, `-readme` is required, otherwise the script will fail.

Reply:
No.  We should provide at least one of -inputfiles and -readme. If -readme provide, the -inputfiles will be ignored and will use -readme to provide inputfiles.

So please revert the change back.

Reply:
Thanks @chunyu3 I'll roll back the changes regarding the generation scripts.